client/main.css

```diff
@@ -656,9 +658,16 @@ audio {
   vertical-align: middle;
   height: 10vh;
 }
+.footer div {
+  margin-bottom: 10px;
+}
 .playlist {
   text-align: center;
-}
+}/*
```

Comment:
As below, if we're doing away with it let's delete altogether instead of commenting out.